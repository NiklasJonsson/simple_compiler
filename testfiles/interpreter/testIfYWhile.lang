int main() {
	int count = 2;
	while(count <= 10) {
		int a = 4;
		int b = 1;
		if(a != count) {
			print(a);
			int a = 3;
			print(a);
		} else { 
			print(b);
			int b = 0;
			print(b);
		}
		print(count);
		count = count + 1;
		print(count);
	}
}