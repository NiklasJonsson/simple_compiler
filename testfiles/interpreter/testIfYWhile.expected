4
3
2
3
4
3
3
4
1
0
4
5
4
3
5
6
4
3
6
7
4
3
7
8
4
3
8
9
4
3
9
10
4
3
10
11
